.circleci/config.yml

```diff
@@ -14,6 +14,9 @@ jobs:
 
       - restore_cache:
           key: v1-macos-gems-{{ checksum "Gemfile" }}
+      - restore_cache:
+          key: v1-homebrew-{{ .BuildNum }}
+          key: v1-homebrew
```

Comment:
And that will never change. Isn't this a source for problems if something in `/usr/local/Homebrew` would/should change?

Reply:
From my previous testing with Circle partial matches produce a cache hit. So even though e.g. in the previous build it will be saved as `v1-homebrew-1234` it will match `v1-homebrew`. I believe we could potentially remove the line above but it's in their example, so I'd rather leave it in. I think I do need to upgrade the `v1` to `v2` though since I previously used the exact key `v1-homebrew` for `save_cache`

`/usr/local/homebrew` is a git repository that brew does `git pull` on when you run it, so if anything ever changes the worst that can happen is that it'll get slower again over time